### PR TITLE
fix deprecations of zeros

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**
The deprecations were incorrect

cc @ChrisRackauckas

### CODE REVIEW

**Comment by DhairyaLGandhi** [comment]
Fixes the case like https://github.com/SciML/DiffEqFlux.jl/pull/592

File: src/deprecations.jl [hunk 1]
```diff
@@ -26,6 +26,7 @@ ones(T::Type, dims...) = Base.ones(T, dims...) # no need for a message
 
 function zeros(dims...)
   Base.depwarn("Flux.zeros(size...) is deprecated, please use Flux.zeros32(size...) or Base.zeros(Float32, size...)", :ones)
```
Comment (CarloLucibello) [inline src/deprecations.jl:28]:
```suggestion
  Base.depwarn("Flux.zeros(size...) is deprecated, please use Flux.zeros32(size...) or Base.zeros(Float32, size...)", :zeros)
```
Not sure what that is

**Comment by CarloLucibello** [comment]
should `ones` get a similar treatment?

**Comment by DhairyaLGandhi** [comment]
I think its also a bit weird that there's an error message about forcing people to use a specific type in `zeros32`.

**Comment by DhairyaLGandhi** [comment]
Merging since this is required by sciml

**Pull request merged**